java/connector-node/risingwave-connector-service/src/main/resources/mysql.properties

```diff
@@ -24,6 +24,4 @@ heartbeat.interval.ms=${debezium.heartbeat.interval.ms:-60000}
 name=${hostname}:${port}:${database.name}.${table.name:-RW_CDC_Sharing}
 # In sharing cdc mode, transaction metadata will be enabled in frontend
 provide.transaction.metadata=${transactional:-false}
-## Pass-through driver properties
-# set connector timezone to UTC(+00:00)
-driver.connectionTimeZone=+00:00
```

Comment:
Didn't get why remove this option.
Timestamp in mysql must be mapped into `timestamptz` in RW, thus we set the `connectionTimeZone` to `+00:00` to eliminate the timezone of upstream MySQL, so that we can store the UTC value in RW.

Reply:
cc @xiangjinwu

Reply:
> Timestamp in mysql must be mapped into timestamptz in RW

It's mapped to UTC by debezium with or without setting the param. Setting connectionTimeZone to +00:00 tells debezium the MySQL timezone is +00:00. If we don't set this, it will be default value `SERVER`, meaning the timezone will be fetched from MySQL server.

Reply:
> If we don't set this, it will be default value SERVER, meaning the timezone will be fetched from MySQL server.

Yes, but we deliberately want Debezium respect the `connectionTimeZone` instead of reading the setting from mysql server. As it may fail to read it from the server:

> The time zone will be queried from the server by default. If this fails, it must be specified explicitly by the database connectionTimeZone MySQL configuration option.

Reply:
> As it may fail to read it from the server

It will only fail when MySQL is running on a system set to an unconventional time zone or can't be recognized by JDBC, which happens rarely. If we want to cover this case, we should make the connectionTimeZone and parameter for user.

Setting this value to a fixed +00:00 is not a correct usage. It should always be set to the mysql server timezone

Reply:
> It will only fail when MySQL is running on a system set to an unconventional time zone or can't be recognized by JDBC, which happens rarely.

I recall that there is a typical case, the `CST` timezone which is ambiguous.

> Setting this value to a fixed +00:00 is not a correct usage. It should always be set to the mysql server timezone

From the mysql [doc](https://dev.mysql.com/doc/connector-j/en/connector-j-connp-props-datetime-types-processing.html#cj-conn-prop_forceConnectionTimeZoneToSession), I think we should pass `connectionTimeZone=+00:00` and `forceConnectionTimeZoneToSession=true`, so that we can guarantee that we are reading the correct UTC value.

Reply:
Yes this is also an option

Reply:
Oh wait I just recalled the reason I decided not to use `forceConnectionTimeZoneToSession`. It's supported only after MySQL 8.0.23

Reply:
> the CST timezone which is ambiguous

I guess the best way to do this is to let users manually set a `debezium.driver. connectionTimeZone`

Reply:
> It's supported only after MySQL 8.0.23

That 8.0.23 version is the version of Connector/J driver, and  we are using 8.0.33 right now.

Reply:
You are right. Let's do connectionTimeZone=+00:00 and forceConnectionTimeZoneToSession=true